src/server/_params.py

```diff
@@ -425,6 +426,17 @@ def extract_dates(key: Union[str, Sequence[str]]) -> Optional[TimeValues]:
         return None
     values: TimeValues = []
 
+    def handle_inequality(part: str):
+        inequality_operator = None
+        for operator in ['<=', '>=', '<', '>']:
+            if part.startswith(operator):
+                inequality_operator = operator
+                part = part[len(operator):]
+                if not part:
+                    raise ValidationFailedException("missing parameter: date after the inequality operator")
+                return (inequality_operator,), part
```

Comment:
nit: using a length-1 tuple here feels weird

later on we're using the tuple type information to engage the inequalities logic instead of the range logic, so it's clearly needed, but it feels unsatisfying regardless

Reply:
hey @rzats do you have any ideas for how we can more clearly indicate that a date parameter value is an inequality? the return value from this function is read in the date span processing at _query.py:45, dates.py:125, and dates.py:147, where inequalities need to be handled differently from other kinds of spans.

Reply:
Yep, I can see what this PR is going for and there might be a better way to handle the inequalities. Right now this is returning a single-element tuple to fit within the following type structure;

```
IntRange: TypeAlias = Union[Tuple[int, int], int]
TimeValues: TypeAlias = Sequence[IntRange]
```

A clean solution would be to add a new type alias called `IntStrRange` which is a union of `Tuple[int, int]`, `int` and a third type - either `str` if we just want to know what the inequality operator is or `Tuple[str, int]` if we want the date too.

`TimeValues` can then be replaced with a `Sequence[IntStrRange]`, and we can then refactor the date span processing functions where `TimeValues` was previously used.